lib/request/rack.rb

```diff
@@ -136,6 +136,7 @@ def if_modified_since
     accessor(:host,         Key.new('SERVER_NAME'))
     accessor(:query_string, Key.new('QUERY_STRING'))
     accessor(:content_type, Key.new('CONTENT_TYPE'), nil)
+    accessor(:cookies,      Key.new('HTTP_COOKIE'), EMPTY_STRING)
```

Comment:
@mbj hmm, originally it used to invoke https://github.com/snusnu/cookie/blob/master/lib/cookie/registry.rb#L17 directly which relies on `header` to be a `String`. I'm pretty free to "break" that assumption tho. I assume it's only fair to provide a way to know wether cookies were present or not and i agree that testing for an empty string would be awkward. That being said, would you just return `nil`? (i assume so)